packages/components/FilePreview/ImagesFullViewModal.tsx

```diff
@@ -52,6 +57,9 @@ export const ImagesFullViewModal: React.FC<ImageFullViewModalProps> = ({
 }) => {
   const [localActiveIndex, setLocalActiveIndex] = useState(activeIndex);
   const { height, width } = useWindowDimensions();
+
+  const imageHeight = Platform.OS !== "web" ? height - 200 : height - 100;
```

Comment:
this one feels quite "magical" and opaque.
Why are exactly these values?

Reply:
@omniwired  
On web 100px is used to set gap of 50px on top and 50px on bottom of the image. 
On app;  200 is required due to  insets. 
I will check if this can be removed completely.